### DISCUSSION TITLE
Smplayer start / pause / stop with command line ?

### DISCUSSION THREAD

**Discussion by stefan-franz** [smplayer-dev/smplayer#466] [Q&A]
Hello, is there a possibility to start playing / pause or stop over command line under Linux Mint?

**Comment by smplayer-dev** [comment]
You can use the option **-actions**, for example `smplayer video.avi -actions pause`

**Comment by stefan-franz** [comment]
Can i only start play (a pre loaded playlist) with any command? Player stands in the full screen mode, has a playlist loaded and waits to "push the play button" - is there a possibility only press play with a command?

**Reply by smplayer-dev** [reply]
Enable the option "Start playback after loading a playlist" in preferences -> playlist.

If SMPlayer is already running you can use this command to start playback:
`smplayer -send-action play`
